python/py-django-countries/Portfile

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           python 1.0
+
+github.setup        SmileyChris django-countries 5.3.2 v
+name                py-django-countries
+categories-append   www
+platforms           darwin
+license             MIT
+maintainers         {mojca @mojca} openmaintainer
+
+description         Provides a country field for Django models
+long_description    A Django application that provides country choices for use with forms, \
+                    flag icons static files, and a country field for models.
+
+python.versions     27 37
+
+checksums           rmd160  cd66038a338d4567784a31504e6bb1ae8ae82b8a \
+                    sha256  13b24040f7c18fd3cd9750b40db9a2f93e6b9eb1c520ef6331839302394d2357 \
+                    size    450501
+
+if {${name} ne ${subport}} {
+    depends_build-append \
+                    port:py${python.version}-setuptools \
+                    port:py${python.version}-pyobjc
```

Comment:
Very minor, but please sort these dependencies.

Reply:
I suspect that pyobjc is not even needed anyway (I only added it because trace mode complained, but I saw it complaining it other modules as well).